forest/src/cli/chain_cmd.rs

```diff
@@ -82,17 +93,24 @@ impl ChainCommands {
 
                 let epoch = tipset.unwrap_or(chain_head.epoch());
 
-                let output_path = match output_path {
-                    Some(path) => path.to_owned(),
-                    None => {
-                        let now = OffsetDateTime::now_utc();
-                        format!(
-                            "forest_snapshot_{}-{}-{}_height_{}.car",
-                            now.year(),
-                            now.month(),
-                            now.day(),
-                            epoch,
-                        )
+                let now = OffsetDateTime::now_utc();
+
+                let month_string = format!("{:02}", now.month() as u8);
+                let year = now.year();
+                let day = now.day();
+                let chain_name = chain_get_name().await.map_err(handle_rpc_err).unwrap();
+
+                let mut vars = HashMap::new();
+                vars.insert("year".to_string(), year.to_string());
```

Comment:
do we need `.to_string()` for keys in the map?

Reply:
Yes, we do.